plugins/arElasticSearchPlugin/lib/arElasticSearchPlugin.class.php

```diff
@@ -280,20 +315,29 @@ public function populate()
 
     foreach ($this->mappings as $typeName => $typeProperties)
     {
-      $camelizedTypeName = sfInflector::camelize($typeName);
-      $className = 'arElasticSearch'.$camelizedTypeName;
+      if (!count($this->excludeTypes) || !in_array(strtolower($typeName), $this->excludeTypes))
```

Comment:
could you remove the !count($this->excludeTypes) part to this if statement? Since excludeTypes will be an empty array and in_array will return false every time anyway? 

it just took my head for a spin a bit when I saw if (!count($this->excludeTypes) || ...) then right after that if (count($this->excludeTypes)) :smiley: